packages/myst-cli/src/process/mdast.ts

```diff
@@ -173,11 +173,11 @@ export async function transformMdast(
 
   const pipe = unified()
     .use(reconstructHtmlPlugin) // We need to group and link the HTML first
-    .use(htmlPlugin, { htmlHandlers }) // Some of the HTML plugins need to operate on the transformed html, e.g. figure caption transforms
     .use(basicTransformationsPlugin, {
       parser: (content: string) => parseMyst(session, content, file),
     })
     .use(inlineExpressionsPlugin) // Happens before math and images!
+    .use(htmlPlugin, { htmlHandlers }) // Some of the HTML plugins need to operate on the transformed html, e.g. figure caption transforms
```

Comment:
Going to test the ordering of this as I have run into a couple issues with having HTML after the "basic transformations".  We could always (I think) move the inlineExpressions higher up.

Reply:
I reordered this so `inlineExpressionsPlugin` happens first, then `htmlPlugin`, then `basicTransformations` - I think that's less risky than moving `htmlPlugin` to after `basic`...

I also rendered the inline expression `value` when `children` were not available to render. I think this is preferable? It means the evaluated code shows up in the output document as opposed to nothing.